### PR TITLE
[WIP] Updating napari's python and Bundled App Installation Instructions

### PR DESCRIPTION
**Pull request by dgmccart**
# Description

Fixing the broken link that points to the install as a bundled app section
Adding new images to the repository for the new content
Adding new verbiage for the bundled app installation copy

## Type of change

- [X] Bug-fix (non-breaking change which fixes an issue)

## Final checklist:
- [X] My PR is the minimum possible work for the desired functionality

### CODE REVIEW

File: docs/tutorials/fundamentals/installation.md
Comment (jaimergp) [inline docs/tutorials/fundamentals/installation.md]:
```suggestion
The first time you open napari you must use the Launchpad, but subsequently, the napari application should show up in Spotlight search.
```

I am not sure if it happens immediately after one usage, or if it's more about macOS learning about your preferences (e.g. it requires more than one click to appear in Spotlight).

File: docs/tutorials/fundamentals/installation.md
Comment (jaimergp) [inline docs/tutorials/fundamentals/installation.md]: A reminder that we haven't tested the All Users installation too much.

File: docs/tutorials/fundamentals/installation.md
Comment (jaimergp) [inline docs/tutorials/fundamentals/installation.md]: I'd avoid mentioning the "Clear the package cache" explicitly here. It's now available (and it's safe to use), but we might change that possibility in the future to deal with reinstallations, updates and other features.

```suggestion
On the next page, click ‘Install’ to start the installation process.
```

**Comment by jaimergp** [comment]
Thanks! Reads nicely and to the point. Instructions are accurate too. I added some comments to avoid shooting ourselves in the foot while we change some features in the next releases, though.

A couple of additional comments about the screenshots. I took those while iterating in the pre-release stages (you can see the version tag). Ideally, we would re-take them for a final release. Also, the ones for Windows are a bit low-quality because I had to use a small VM with zoomed-in canvas, hence the pixelation. 

Also, I believe we can link to the last release Downloads page with this hyperlink:

https://github.com/napari/napari/releases/latest

**Comment by chili-chiu** [comment]
closes napari/docs#15

**Review by jni — CHANGES_REQUESTED** [review]
Thanks @dgmccart!

Unfortunately our docs contribution process is a bit in-flux, but in no case do we want to merge documentation images into the repo, as this bloats the repo forever (because they will be in the history even if we remove them later). Currently, they need to be added to git-lfs, using something along the lines of:

```
brew install git-lfs
git lfs install
git add docs/images/*.png
```

I don't know how to move files already committed to lfs, maybe @melissawm has suggestions there.

Regarding the bundle install instructions, I thought we had decided to hold off on promoting it until we had an in-place update process? Apologies if this has not been communicated widely enough... (Or if I have misunderstood something.)

**Comment by chili-chiu** [comment]
> Thanks @dgmccart!
> 
> Unfortunately our docs contribution process is a bit in-flux, but in no case do we want to merge documentation images into the repo, as this bloats the repo forever (because they will be in the history even if we remove them later). Currently, they need to be added to git-lfs, using something along the lines of:
> 
> ```
> brew install git-lfs
> git lfs install
> git add docs/images/*.png
> ```
> 
> I don't know how to move files already committed to lfs, maybe @melissawm has suggestions there.
> 
> Regarding the bundle install instructions, I thought we had decided to hold off on promoting it until we had an in-place update process? Apologies if this has not been communicated widely enough... (Or if I have misunderstood something.)

Thanks @jni for pointing these out :) We should've made the intention of this PR clearer. This is a WIP/draft for updating the installation guide and hope to get feedback from the community (especially @jaimergp and @goanpeca) to check the accuracy/structure. 
Totally agree this should not be merged until the rest of the bundled app implementation is done. 
 
We are aware of the lfs situation and would love to get some help once it's closer to the merge stage :)

**Comment by melissawm** [comment]
Should we move this over to https://github.com/napari/docs, even if it is a WIP?

**Comment by psobolewskiPhD** [comment]
Moving it is slightly tricky, see: 
https://github.com/napari/napari/pull/5240#issuecomment-1282076004

This is a very important PR though, definitely need to get the new bundle install into the docs, it's something people are not super aware of (recently came up on image.sc).
Also, in my opinion, the bundle should get mentioned on the napari.org homepage https://napari.org/stable/
Most people probably look at those install instructions and don't dig deeper to Usage > Getting started. Frankly, I'm not sure I'd look for install instructions there: `Usage` and `Getting started` imply already having napari installed...

However, if this PR needs updating, then it may be easier to just start from scratch?

**Pull request closed without merging**
